gen/AArch64Compile_gen.ml

```diff
@@ -2194,12 +2194,12 @@ module Make(Cfg:Config) : XXXCompile_gen.S =
         let cs = [Instruction (lsri64 r1 r 12)] in
         r1,init,cs,st
 
-    let emit_shootdown_sync dom op r =
-      pseudo
-        (I_FENCE (DSB(dom,FULL))::
-        I_TLBI(op,r)::I_FENCE (DSB(dom,FULL))::[])
+    let emit_shootdown_sync dom op r csr =
+      pseudo ([I_FENCE (DSB(dom,FULL))])
+        @ csr
+        @ pseudo([I_TLBI(op,r); I_FENCE (DSB(dom,FULL))])
 
-    let emit_shootdown_nosync op r = pseudo (I_TLBI(op,r)::[])
+    let emit_shootdown_nosync op r csr = csr @ pseudo ([I_TLBI(op,r)])
```

Comment:
We are we doing this? The order of the LSR with respect to the DSB doesn't matter. I don't understand the point of commit e5f2312adf7fcf2d877dd9f2406877e57463b55d

Reply:
This is a cosmatic change that @murzinv suggested. Also I have go through the `catalogues`, the sequence of `DSB ISH; LSR; TLBI; DSB ISH` is more common. This commit basic aligns to this more common pattern. Previously, it was `LSR; DSB ISH; TLBI; DSB ISH`. Notes that the old sequence has `LSR` before the fence.